scripts/zones/Windurst_Woods/npcs/Anillah.lua

```diff
@@ -32,7 +32,10 @@ function onEventUpdate(player, csid, option)
 end
 
 function onEventFinish(player, csid, option)
-    if csid == 10015 and option == 1 then
+    if
+        csid == 10015 and
```

Comment:
> [Try not to exceed 120 chars width. Exceptions will occur, but try.](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#general-code-guidlines-all-languages)

.

> [-- Long or many multiple conditions - Correct ✔️](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#formatting-conditional-blocks)


Notice how this was not long nor was it many. Already talked to you about this before. so ... REEEEEEEEEEEEEEEEeee :stuck_out_tongue: